analyzer/analyzer.go

```diff
@@ -44,12 +62,31 @@ func New() *analysis.Analyzer {
 		Run:      n.run,
 		Requires: []*analysis.Analyzer{inspect.Analyzer},
 	}
-	r.Flags.BoolVar(&n.intConv, "int-conversion", true, "optimizes even if it requires an int or uint type cast")
-	r.Flags.BoolVar(&n.errError, "err-error", false, "optimizes into err.Error() even if it is only equivalent for non-nil errors")
-	r.Flags.BoolVar(&n.errorf, "errorf", true, "optimizes fmt.Errorf")
-	r.Flags.BoolVar(&n.sprintf1, "sprintf1", true, "optimizes fmt.Sprintf with only one argument")
+	r.Flags.BoolVar(&n.intFormat.enabled, "integer-format", true, "enable/disable optimization of integer formatting")
+	r.Flags.BoolVar(&n.intFormat.intConv, "int-conversion", true, "optimizes even if it requires an int or uint type cast")
+	r.Flags.BoolVar(&n.errFormat.enabled, "error-format", true, "enable/disable optimization of error formatting")
+	r.Flags.BoolVar(&n.errFormat.errError, "err-error", false, "optimizes into err.Error() even if it is only equivalent for non-nil errors")
+	r.Flags.BoolVar(&n.errFormat.errorf, "errorf", true, "optimizes fmt.Errorf")
+	r.Flags.BoolVar(&n.boolFormat, "bool-format", true, "enable/disable optimization of bool formatting")
+	r.Flags.BoolVar(&n.hexFormat, "hex-format", true, "enable/disable optimization of hex formatting")
+	r.Flags.BoolVar(&n.strFormat.enabled, "string-format", true, "enable/disable optimization of string formatting")
```

Comment:
Following my [suggestion](https://github.com/catenacyber/perfsprint/pull/41#discussion_r1935362061) to invert the flags


```suggestion
	r.Flags.BoolVar(&n.intFormat.deactivated, "integer-format", false, "activate/deactivate optimization of integer formatting")
	r.Flags.BoolVar(&n.intFormat.intConv, "int-conversion", true, "optimizes even if it requires an int or uint type cast")
	r.Flags.BoolVar(&n.errFormat.deactivated, "error-format", false, "activate/deactivate optimization of error formatting")
	r.Flags.BoolVar(&n.errFormat.errError, "err-error", false, "optimizes into err.Error() even if it is only equivalent for non-nil errors")
	r.Flags.BoolVar(&n.errFormat.errorf, "errorf", true, "optimizes fmt.Errorf")
	r.Flags.BoolVar(&n.boolFormat, "bool-format", true, "activate/deactivate optimization of bool formatting")
	r.Flags.BoolVar(&n.hexFormat, "hex-format", true, "activate/deactivate optimization of hex formatting")
	r.Flags.BoolVar(&n.strFormat.deacivated, "string-format", false, "activate/deactivate optimization of string formatting")
```

Here the wording, or the flags name might need to be adapted.

Reply:
done something